examples/01c-Getting-started-Pytorch.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>Let's do not calculate loss in the beginning, I havent seen that in another pipeline/example?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>removed</p>